executor/ddl_test.go

```diff
@@ -134,6 +134,7 @@ func (s *testSuite6) TestCreateTable(c *C) {
 			c.Assert(req.GetRow(0).GetString(1), Equals, "double")
 		}
 	}
+	c.Assert(rs.Close(), IsNil)
```

Comment:
We usually do not care about the result of Close(). But yeah, it is not bad to check it, either. Nice Work.